notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>General comment: Search for "will" throughout, and update those to use present tense (https://developers.google.com/style/tense). For example:</p><ul><li>"will return" --&gt; "returns"</li><li>"you will X" --&gt; "you X" or "X" (where X is a verb)</li></ul><p> </p><p>General comment: Remove "we" (throughout)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>